renovate.json

```diff
@@ -1,11 +1,3 @@
 {
-  "extends": [":base"],
-  "ignoreDeps": ["expect"],
-  "automerge": true,
-  "major": {
-    "automerge": false
-  },
-  "masterIssue": true,
-  "timezone": "America/Los_Angeles",
-  "schedule": ["on friday"]
+  "automerge": true
```

Comment:
Is this config and schedule required to trigger `renovate`? Or has that all been moved into Github Apps/Actions context? And if so can the config be removed entirely? (I am a beginner at Renovate).

Reply:
I didn't want to totally remove renovate.json because I still want it to run, but I don't think the "automerge" part is necessary anymore. I expect I'll have to debug this a bit as we watch new dependency bot PRs come in though, so we'll see!